src/components/Reports/PartnerGivingAnalysisReport/Table/Table.tsx

```diff
@@ -126,12 +123,18 @@ export const PartnerGivingAnalysisReportTable: FC<
                   />
                 </TableCell>
                 <TableCell>
-                  <ContactName
-                    onClick={() => onClick(contact.id)}
-                    onMouseEnter={preloadContactsRightPanel}
-                  >
-                    {contact.name}
-                  </ContactName>
+                  <NextLink href={contactUrl} passHref>
+                    <Link
+                      component="a"
```

Comment:
I don't think this `component="a"` is necessary. And here's another link that's always underlined instead of on hover like most of the links in the app.

Maybe we should add `defaultProps: { underline: 'hover' }` to `MuiLink` in the theme to help with consistency.